flexget/plugins/sites/lostfilm.py

```diff
@@ -69,7 +67,7 @@ def on_task_input(self, task, config):
             return
         try:
             rss = feedparser.parse(config['url'])
-        except:
+        except Exception:
```

Comment:
Hmm, they are a bit different, `except Exception` doesn't catch SystemExit and KeyboardInterrupt I think it might be a bit better like this than the raw except.

Reply:
I see. Seems unlikely to ever trigger, but fine with me.

Reply:
Yep, extremely unlikely to make a difference in practice, but in theory we don't want to block KeyboardInterrupts from exiting the program.